src/Illuminate/Support/Facades/Date.php

```diff
@@ -0,0 +1,267 @@
+<?php
+
+namespace Illuminate\Support\Facades;
+
+use DateTimeInterface;
+use InvalidArgumentException;
+use Illuminate\Support\Carbon;
+
+/**
+ * @method static \Illuminate\Support\Carbon                         create($year = 0, $month = 1, $day = 1, $hour = 0, $minute = 0, $second = 0, $tz = null)                       Create a new Carbon instance from a specific date and time.
+ *                                                                                                                                                                                  If any of $year, $month or $day are set to null their now() values will
+ *                                                                                                                                                                                  be used.
+ *                                                                                                                                                                                  If $hour is null it will be set to its now() value and the default
```

Comment:
What's with all the extra whitespace?

Reply:
This is for alignment, it's get from auto-generated doc of the Carbon factory.

Reply:
Longest type is `\Symfony\Component\Translation\TranslatorInterface` so it pad whitespace on smaller types and longest method is `createSafe($year = null, $month = null, $day = null, $hour = null, $minute = null, $second = null, $tz = null)` so it pad whitespace on other methods.

Reply:
I see other faces does not describe methods and does not align anything, would you like the same for this Date facade?

Reply:
I know, I will compact it then put some links to the reference docs

Reply:
@GrahamCampbell done

Reply:
@GrahamCampbell Verbose documentation removed.